migration_tools_example/config/install/migrate_plus.migration_tools_example_drupal_import.yml

```diff
@@ -5,50 +5,74 @@ migration_group: migration_tools_example
+
+  # Migration Tools Settings, can be used with any source plugin.
+  migration_tools:
+    # An array of sources to process
     -
-      # Finds the body by plucking the .field-name-body field.
-      name: body
-      label: 'Body'
-      obtainer: ObtainBody
-      # Run Obtainer after modifiers are run.
-      after_modify: TRUE
-      jobs:
-        - { job: 'addSearch', method: 'pluckSelector', arguments: [ '.field-name-body', 1 ] }
+      source: url # Source field
+      source_type: url # What the source field contains - 'url' or 'html'
 
-  # Modifiers allow for addPreModifier or addModifier.
-  # Methods are in ModifyHtml class
-  modifiers:
-    - { modifier: addPreModifier, method: convertBaseHrefLinks }
-    - { modifier: addPreModifier, method: fixNamedAnchors }
-    - { modifier: addPreModifier, method: removeFaultyImgLongdesc }
-    - { modifier: addModifier, method: removeSelectorAll, arguments: [ '.topnavlinks' ] }
-    - { modifier: addModifier, method: removeEmptyTables }
-    - { modifier: addModifier, method: removeElements, arguments: [ '.pagedate', 'head' ] }
+      # Source operations are performed on the HTML prior to initializing the queryPath.
+      # Valid source operations are in SourceModifierHtml.
+      source_operations:
+        -
+          operation: modifier
+          modifier: basicCleanup
+      # Fields define jobs to retrieve the data from the DOM and what field they will set in row.
+      fields:
+        title:
+          # Finds the title by plucking the .page-subtitle field.
+          obtainer: ObtainTitle
+          jobs:
+            -
+              # Any number of jobs can be configured using find, or pluck to remove it from the DOM.
+              # Jobs will continue until a result is found
+              job: 'addSearch'
+              method: 'pluckSelector'
+              arguments:
+                - .page-subtitle
+                - '1'
+        body:
+          # Finds the body by plucking the .field-name-body field.
+          obtainer: ObtainBody
+          jobs:
+            -
+              job: 'addSearch'
+              method: 'pluckSelector'
+              arguments:
+                - .field-name-body
+                - '1'
+      # DOM Operations performs the field jobs and applied modifiers in order.
+      dom_operations:
+        -
+          operation: get_field # Operation to perform - 'get_field' or 'modifier'
+          field: title # Field from above to get (run jobs)
+        -
+          operation: modifier
+          modifier: removeSelectorAll
+          arguments:
+            - '.topnavlinks'
+        -
+          operation: modifier
+          modifier: removeEmptyTables
+        -
+          operation: modifier
+          modifier: removeSelectorAll
+          arguments:
+            - '.pagedate'
+            - 'head'
+        -
+          # Get the body field after above modifiers have run.
+          operation: get_field
+          field: body
```

Comment:
I really like how readable that was.    It was clear to see what was going on.  Nice example